lib/datadog/tracing.rb

```diff
@@ -0,0 +1,118 @@
+module Datadog
+  # Datadog APM tracing public API.
+  #
+  # The Datadog teams ensures that public methods in this module
+  # only receive backwards compatible changes, and breaking changes
+  # will only occur in new major versions of `ddtrace`.
+  module Tracing
+    class << self
+      # TODO: copy docs from Datadog::Tracer#trace
+      def trace(*args, &block)
+        tracer.trace(*args, &block)
+      end
+      ruby2_keywords :trace if respond_to?(:ruby2_keywords, true)
```

Comment:
The current signature of trace is `def trace(name, options = {})`.

Since we're moving stuff here, is it perhaps time to just make this `def trace(name, **options, &block)` as well, and thus avoiding the need for `ruby2_keywords`?

Reply:
Not sure... I think I might actually make this a `delegates_to`.